framework/CodeInterfaces/Prescient/PrescientCodeInterface.py

```diff
@@ -145,22 +145,32 @@ def _readBusData(self, filename):
         line = line.strip()
         if first:
           first = False
-          if not line.startswith("Date,Hour,Bus,"): # != "Date,Hour,Bus,Shortfall,Overgeneration,LMP,LMP DA":
+          if line.startswith("Date,Hour,Minute,Bus,"):
+            hasMinute = True
+          elif line.startswith("Date,Hour,Bus,"):
+            #line sorta looks like: "Date,Hour,Bus,Shortfall,Overgeneration,LMP,LMP DA"
+            hasMinute = False
+          else:
             assert False, "Unexpected first line of bus detail:" + line
             a = 1/0 #because debug might be disabled
```

Comment:
Is this line correct?

Reply:
Hm, it is at least sorta correct, but there might be a better way to signal an error from a Code Interface. I should look.

Reply:
How about I change this to something like:
```
raise IOError("Unexpected first line of bus detail in Prescient Code Interface:" + line)
```

Reply:
Changed.